features/eidas_user_journeys.feature

```diff
@@ -87,3 +87,15 @@ Feature: eIDAS user journeys
     And they select IDP "Stub IDP Demo"
     And they login as "stub-country" with "rsassa-pss" signing algorithm
     Then they should be successfully verified
+
+  @Eidas
+  Scenario: EID-1053, User fails to authenticate presses back button and tries again
+    Given the user is at Test RP
+    And they start an eIDAS journey
+    And they select eIDAS scheme "Stub IDP Demo"
+    Then they should be at IDP "Stub Country"
+    Given the IDP fails to authenticate user
+    Then they should arrive at the Failed country sign in page
+    Given they go back to the country picker
+    And they select eIDAS scheme "Stub IDP Demo"
+    Then they should be at IDP "Stub Country"
```

Comment:
Can we extend this test successfully sign in with a country after going back to the picker page?